sources/netdog/src/networkd/devices/vlan.rs

```diff
@@ -63,13 +65,17 @@ impl NetworkFileCreator for NetworkDVlan {
             routes,
             device: _, // device and id aren't used in .network files
             id: _,
+            mtu,
         } = self;
 
         let mut network = NetworkBuilder::new_vlan(name.clone());
         network.with_dhcp(dhcp4.clone(), dhcp6.clone());
         maybe_add_some!(network, with_static_config, static4);
         maybe_add_some!(network, with_static_config, static6);
         maybe_add_some!(network, with_routes, routes);
+        if let Some(m) = mtu {
+            network.with_mtu(*m);
+        }
```

Comment:
Same suggestion for the other occurrences of this pattern:
```suggestion
    maybe_add_some!(network, with_mut, mtu);
```